Week0/wantkdd/mission1/mission1.html

```diff
@@ -28,19 +28,22 @@
         gap: 30px;
         color: white;
       }
+      li{
+        list-style: none;
+      }
     </style>
   </head>
   <body>
     <nav>
       <div class="logo">
         <img src="UMC_logo.png" alt="로고" />
       </div>
-      <div class="menu">
-        <div>소개</div>
-        <div>프로젝트</div>
-        <div>Q&A</div>
-        <div>지원하기</div>
-      </div>
+      <ul class="menu">
+        <li>소개</li>
+        <li>프로젝트</l>
+        <li>Q&A</l>
+        <li>지원하기</l>
```

Comment:
여기 오타가 있는 것 같아서, 이것만 마지막으로 수정해주세요..!!
```suggestion
        <li>소개</li>
        <li>프로젝트</li>
        <li>Q&A</li>
        <li>지원하기</li>
```